index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+
+</head>
+<body>
+    <div id="top_bar"> 
+        <div id="top1">
+          <img  src="img/top_bar1.png" alt="" >
+        </div >
```

Comment:
regarding the div that contains an image of both the hamburger icon and youtube icon, the hamburger icon and youtube icon are supposed to be separate elements and each should be wrapped by button element  since they are supposed to act as a button that the user uses in order to interact with the website 😄